sycl/test-e2e/DeviceCodeSplit/aot-accelerator.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: opencl-aot, accelerator
 
-// RUN: %clangxx -fsycl -fsycl-device-code-split=per_source -fsycl-targets=spir64_fpga -I %S/Inputs -o %t.out %S/split-per-source-main.cpp %S/Inputs/split-per-source-second-file.cpp \
+// RUN: %clangxx -fsycl -fsycl-device-code-split=per_source -fsycl-targets=spir64_x86_64 -I %S/Inputs -o %t.out %S/split-per-source-main.cpp %S/Inputs/split-per-source-second-file.cpp \
```

Comment:
This seems wrong. I think we need to remove this test. Thanks

Reply:
An equivalent test already exists - so this has been removed.